roles/commcare_analytics/templates/superset/superset_config.py.j2

```diff
@@ -115,3 +116,38 @@ LANGUAGES = {
 # CommCare Analytics extensions
 FLASK_APP_MUTATOR = flask_app_mutator
 CUSTOM_SECURITY_MANAGER = oauth.CommCareSecurityManager
+
+# Any project which wants to add custom images to their dashboards must have
+# the url of the image source added here, e.g. "https://mycoolimagesite.com"
+
+TALISMAN_CONFIG = {
```

Comment:
This config was copied from the superset codebase and tweaked slightly for our own uses. 

The TALISMAN_ENABLED flag was set to a default `True` value since superset 3 for security reasons (this flag enables content security policy checking). Likewise, the TALISMAN_CONFIG is the policy configuration.

Reply:
I have tried to simply edit the policies we're interested in, but it didn't seem to work quite right. I suspect whatever we set overrides the default variable instead of appending to it.